config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
+  devise_for :users, skip: %i(registrations sessions confirmations passwords), controllers: { omniauth_callbacks: "users/omniauth_callbacks" }
```

Comment:
cho vào scope locale đi em
trùng với line 4 phía dưới

Reply:
Dạ devise không hổ trợ `Devise does not support scoping OmniAuth callbacks` nên em bỏ ngoài và skip để không lặp lại ạ.